Remotion/ObjectBinding/Web/UI/Controls/BocEnumValueImplementation/Rendering/BocEnumValueRenderer.cs

```diff
@@ -169,7 +169,10 @@ private ListControl GetListControl (BocEnumValueRenderingContext renderingContex
       var isDropDownList = renderingContext.Control.ListControlStyle.ControlType == ListControlType.DropDownList;
 
       if (isRadioButtonList)
+      {
         listControl.Attributes.Add (HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.RadioGroup);
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, "remotion-themed");
```

Comment:
There's a property / constant for this. See `WebButton.CssClassThemed` for an example